builds/jazz-build-module/environment-deployment-metadata-loader.groovy

```diff
@@ -38,13 +39,14 @@ def initialize(serviceConfig, configLoader, scmMdule, branch, buildUrl, buildId,
 }
 
 def getEnvironmentLogicalId() {
-	if (g_environment_logical_id == null && g_service_config['domain'] != "jazz") {
+	if ((g_environment_logical_id == null || g_environment_details == null) 
+								&& g_service_config['domain'] != "jazz") {
```

Comment:
Why do we need to filter based on domain? This should work for all services irrespective of the domain.

Reply:
Initially it used. Now not using. So removed